pkg/solana/relay.go

```diff
@@ -57,6 +58,11 @@ func (r relayer) Healthy() error {
 	return nil
 }
 
+type Transmitter interface {
+	Sign(msg []byte) ([]byte, error)
+	PublicKey() solana.PublicKey
+}
+
```

Comment:
:+1:

Reply:
Should be `Signer` as it just signs and has an id, doesn't transmit anything.

Np now, we'll tweak things like this at a later time.

Reply:
Cosmos SDK interface example: https://github.com/cosmos/cosmos-sdk/blob/779c7e2e174b1dff4f71329a5662c4b8806d36dd/crypto/keyring/keyring.go#L109